Xamarin.Forms.Core/Device.cs

```diff
@@ -16,15 +16,15 @@ public static class Device
 
 		public static TargetPlatform OS { get; internal set; }
 
-		internal static DeviceInfo Info
+		public static DeviceInfo Info
```

Comment:
Yap there's 2 things we need to discuss the proper API to be exposed, naming etc.. Also we need to figure what would we want to expose if a orientation change, a size change etc..